azext_iot/tests/iothub/test_iot_messaging_int.py

```diff
@@ -106,6 +106,36 @@ def test_uamqp_device_messaging(self):
             checks=self.is_empty(),
         )
 
+        utf_32_encoding = "utf-32"
+        string_payload = "Test payload encoding decoding"
+
+        # Send C2D Message with UTF-32 encoding
+        self.cmd(
+            """iot device c2d-message send -d {} -n {} -g {} --data '{}' --cid {} --mid {} --ct {} --expiry {}
```

Comment:
These tests are already covered in the `dev` branch https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/tests/iothub/messaging/test_iothub_c2d_messages_int.py

Reply:
I will take care of this in the merge

Reply:
This probably won't come up in the merge because new c2d send tests are in a different file

Reply:
Please keep this unresolved until its taken care of so we can track what work remains.

Reply:
Okay I will remove the test in the next iteration